scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -21,13 +21,11 @@ workflow GvsCreateFilterSet {
     String? service_account_json_path
     Int? SNP_VQSR_max_gaussians_override = 6
     Int? SNP_VQSR_mem_gb_override
+    # this is the minimum number of samples where the SNP model will be created and applied in separate tasks
+    # (SNPsVariantRecalibratorClassic vs. SNPsVariantRecalibratorCreateModel and SNPsVariantRecalibratorScattered)
+    Int snps_variant_recalibration_threshold = 5000
```

Comment:
I agree with the change, but it might be worth noting that for WARP classic this is done with 20k. Probably not necessary since it's purely implementation, but I know we've been trying to stay as true to WARP as possible